pkg/master/reconcilers/lease.go

```diff
@@ -82,7 +82,7 @@ func (s *storageLeases) UpdateLease(ip string) error {
 			},
 		}
 
-		leaseTime := uint64(s.leaseTime)
+		leaseTime := uint64(s.leaseTime / time.Second)
```

Comment:
Worth commenting that leaseTime has to be in seconds and hence the division?

Reply:
Yep, will add a comment